web/components/home/Home.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { TextInput } from '../input/text';
+import { PrimaryButton } from '../input/buttons';
+
+import sun from '../../assets/sun.svg';
+
+import skyline from '../../assets/skyline.svg';
+
+import panelBear from '../../assets/panel-bear.svg';
+import panelTrain from '../../assets/panel-train.svg';
+
+import bearCircle from '../../assets/bear-circle.svg';
+
+const getImageDiv = (alt, src, scale) => (
+  <img
+    className={scale ? 'scale-width-phablet' : ''}
+    alt={alt}
+    src={src} />
+);
+
+const rowStyle = `
+  flex jc-between ai-center dir-row
+  pad-sides-ultra pad-bottom-giga scale-pad-sides-tablet`;
+
+const Home = () => (
+  <div className="homepage overflow-hidden">
+    <div className="flex jc-between ai-center pad-top-ultra pad-sides-ultra scale-pad-sides-tablet homepage-scale-top">
+      <div>
+        <h1 className="scale-h1-tablet">nwHacks 2019</h1>
+        <p className="primary">
+          Western Canada&apos;s largest hackathon<br />
+          January 26-27, 2019 @ the University of British Columbia
+        </p>
+        <p className="secondary">
+          Get notified when registration opens!
+        </p>
+        <div className="homepage-email-registration flex ai-end">
+          <TextInput
+            placeholder="hacker@email.com" />
```

Comment:
![email-error](https://user-images.githubusercontent.com/21287547/44304280-6be67300-a30c-11e8-852e-9caa49f25a01.png)

Probably need to define `onChange` function here? And save the email as a state of this component (this could be addressed in another PR)?

Reply:
righto ill add a placeholder for now and open a ticket for hooking it up with actual functionality